web/src/pages/ListLogSources/LogSourceCards/LogSourceCard.tsx

```diff
@@ -67,13 +67,17 @@ const LogSourceCard: React.FC<LogSourceCardProps> = ({ source, children, logo })
       <GenericItemCard.Logo src={logo} />
       {!isCreatedByPanther && <LogSourceCardOptions source={source} />}
       <GenericItemCard.Body>
-        <Link
-          as={RRLink}
-          to={urls.logAnalysis.sources.edit(source.integrationId, sourceType)}
-          cursor="pointer"
-        >
-          <GenericItemCard.Heading>{source.integrationLabel}</GenericItemCard.Heading>
-        </Link>
+        {!isCreatedByPanther ? (
+          <GenericItemCard.Heading>
+            <Link as={RRLink} to={urls.logAnalysis.sources.edit(source.integrationId, sourceType)}>
+              {source.integrationLabel}
+            </Link>
+          </GenericItemCard.Heading>
+        ) : (
+          <GenericItemCard.Heading>
+            <Text color="blue-200">{source.integrationLabel}</Text>
```

Comment:
This is not a link anymore, so IMO it shouldn't have the color of a link in order not to confuse the user.

I propose a `teal-300` that will make it "stand out" as differentt, but not mark it as link
```suggestion
            <Text color="teal-300">{source.integrationLabel}</Text>
```